services/meta/service_test.go

```diff
@@ -567,83 +559,40 @@ func TestMetaService_Subscriptions_Create(t *testing.T) {
 	}
 
 	// Create a subscription
-	if res := c.ExecuteStatement(mustParseStatement(`CREATE SUBSCRIPTION sub0 ON db0."default" DESTINATIONS ALL 'udp://example.com:9090'`)); res.Err != nil {
-		t.Fatal(res.Err)
+	if err := c.CreateSubscription("db0", "default", "sub0", "ALL", []string{"udp://example.com:9090"}); err != nil {
+		t.Fatal(err)
 	}
 
 	// Re-create a subscription
-	if res := c.ExecuteStatement(mustParseStatement(`CREATE SUBSCRIPTION sub0 ON db0."default" DESTINATIONS ALL 'udp://example.com:9090'`)); res.Err == nil {
-		t.Fatal(res.Err)
+	if err := c.CreateSubscription("db0", "default", "sub0", "ALL", []string{"udp://example.com:9090"}); err == nil || err.Error() != `subscription already exists` {
+		t.Fatalf("unexpected error: %s", err)
 	}
 
-	res := c.ExecuteStatement(mustParseStatement(`SHOW SUBSCRIPTIONS`))
-	if res.Err != nil {
-		t.Fatal(res.Err)
-	} else if got, exp := len(res.Series), 1; got != exp {
-		t.Fatalf("unexpected response.\n\ngot: %d series\nexp: %d\n", got, exp)
-	}
+	// FIXME(benbjohnson): Convert to metadata reads.
```

Comment:
Weird. I swear I removed these sections. This needs to be tested at the `QueryExecutor` level so I wasn't going to rewrite them here.